public/app/ui/pages/about/AboutPage.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'ui/responsive'
+        ],
+        function (responsive) {
+            return function () {
+                this.ready = function (element, callback) {
+                    responsive.update();
+                    callback();
```

Comment:
These hooks are all asynchronous, so that things like AJAX calls can happen in them, so this call to `callback()` is necessary or the hook will never complete.